source/connection-options.txt

```diff
@@ -0,0 +1,592 @@
+.. _java-rs-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
```

Comment:
I left it as one because the pymongo docs did it as 1--do you think that 2 would be better? I was worried there would be too many subsections if it was at 2 depth

Reply:
Ah okay, personally I'd appreciate seeing all the subsections in the TOC but up to you! I don't think it's too too many